frontend/cmd/cmd.go

```diff
@@ -103,7 +105,11 @@ func (opts *FrontendOpts) Run() error {
 	}
 
 	// Initialize Clusters Service Client
+
 	conn, err := sdk.NewUnauthenticatedConnectionBuilder().
+		TransportWrapper(func(r http.RoundTripper) http.RoundTripper {
+			return otelhttp.NewTransport(r)
```

Comment:
Consider setting correlation and requestID as attributes and put it into the baggage.

Reply:
Maybe in the future if needed.